libs/aws/langchain_aws/llms/bedrock.py

```diff
@@ -130,7 +130,10 @@ def prepare_input(
                 input_body["prompt"] = _human_assistant_format(prompt)
                 if "max_tokens_to_sample" not in input_body:
                     input_body["max_tokens_to_sample"] = 1024
-        elif provider in ("ai21", "cohere", "meta", "mistral"):
+        elif provider in ("ai21", "cohere", "mistral"):
+            input_body["prompt"] = prompt
+        elif provider == "meta":
+            input_body = dict()
```

Comment:
This excludes all input body from all model arguments, is that an expected change? Should those be passed in a separate key?

Reply:
In AWS Bedrock, It says that that the body for a API request to Llama 3 should be:
"body": "{\"prompt\":\"this is where you place your input text\",\"max_gen_len\":512,\"temperature\":0.5,\"top_p\":0.9}" 
(the field "prompt" is mandatory, others are optionals)

Without "input_body = dict()" I had the error:
"required key [prompt] not found#: extraneous key [stop_sequences] is not permitted"
Because in the input body remains the field "stop_sequence", which is a not-expected key for Llama 3.